docs/distribution/distribution.md

```diff
@@ -12,52 +12,130 @@ be structured and how it should behave.
 
 <!-- MarkdownTOC levels="2,3" autolink="true" -->
 
-- [Enso Home Layout](#enso-home-layout)
 - [Universal Launcher Script](#universal-launcher-script)
+- [Enso Distribution Layout](#enso-distribution-layout)
+  - [Portable Enso Distribution Layout](#portable-enso-distribution-layout)
+  - [Installed Enso Distribution Layout](#installed-enso-distribution-layout)
+  - [Converting a Portable Distribution into Installed](#converting-a-portable-distribution-into-installed)
 - [Layout of an Enso Version Package](#layout-of-an-enso-version-package)
   - [Standard Library](#standard-library)
   - [Resolvers](#resolvers)
 
 <!-- /MarkdownTOC -->
 
-## Enso Home Layout
-All of Enso's binaries, packages, etc. are installed into a directory in
-the user's home directory. For macOS and linux distributions that's `~/.enso`,
-by default. The distribution is fully portable, so it never makes any
-assumptions about actually being placed in any particular directory.
+## Universal Launcher Script
+The [universal launcher script](./launcher.md) should be able to launch the
+proper version of Enso executable based on the version specified in the project
+being run, or use the default version if none specified. It should also be able
+to launch other Enso components, provided as
+[plugins](./launcher.md#running-plugins).
+
+**Note**
+This launcher is under development. Until it is in a ready-to-use state, the
+Enso version packages provide simple launcher scripts in the `bin` directory of
+that package. They will be removed when the universal launcher matures.
```

Comment:
This should be a `>` block. Why would they be removed entirely? They still could help the launcher start the enso artefacts, no?

Reply:
@kustosz told me that they should be removed after the universal launcher script matures, as they were more of a hack.

The launcher should most likely handle launching itself, as it must have fine-grained control on the used JVM and for example, additional JVM parameters.